apps/web/src/components/editor/index.tsx

```diff
@@ -841,6 +841,7 @@ function UnlockNoteView(props: UnlockNoteViewProps) {
             note: session.note,
             saveState: SaveState.Saved,
             sessionId: `${Date.now()}`,
+            tags: session.tags,
```

Comment:
Fetch the note's tags here directly instead of passing it through the locked session.